src/components/chat-view-container/chat-view.tsx

```diff
@@ -201,10 +201,16 @@ export class ChatView extends React.Component<Properties, State> {
 
   renderMessages() {
     const messagesByDay = this.getMessagesByDay();
-    const groupDays = Object.keys(messagesByDay).sort((a, b) => (a > b ? 1 : -1));
+    const filteredMessagesByDay = filterAdminMessages(messagesByDay);
```

Comment:
I think it might make more sense to filter this at a higher level component but this is fine for now.

Reply:
Yeah if we want to handle at a higher level, the `chat-view-container`, `get messages()` getter may be the place to filter. Shall note to move that. Thanks Dale.